tests/__snapshots__/dist.test.ts.snap

```diff
@@ -28,9 +28,9 @@ exports[`Build distribution should match a specific file output: dist-output 1`]
   "./build/fonts/e0f30d27bec7c8dbb6fc.woff2",
   "./build/fonts/e4faf134e53e0f37296b.woff2",
   "./build/fonts/ed292ec87cce0d76fd3c.woff2",
-  "./build/images/032b764e59d78db07b56.svg",
+  "./build/images/133911d429584887f5c7.svg",
+  "./build/images/4ca682d556b7bf59f5c6.svg",
```

Comment:
That change was also made in #474 , which is now merged. Please pull and rebase.

Reply:
done